Q2.txt

```diff
@@ -0,0 +1,31 @@
+Answer 2 
+
+let age = 25;
+switch(age)
```

Comment:
No need of using so many cases
you can write conditions in case
eg 
```
switch(true){
 case  age<=12:
   console.log("child")
}
```

Reply:
Thanks for the info sir.